action.yml

```diff
@@ -34,6 +34,11 @@ inputs:
       deps.
     required: false
     default: "true"
+  load-poetry-path:
+    description: >
+      Set to "true" to add the path to the poetry executable to the PATH.
+    required: false
+    default: "false"
```

Comment:
Why not default to `true` as a sane default?

Reply:
Mostly wanted to avoid changing the action default behaviour. WDYT?